app/scripts/metamask-controller.js

```diff
@@ -781,7 +781,7 @@ export default class MetamaskController extends EventEmitter {
     this.snapExecutionService =
       this.opts.overrides?.createSnapExecutionService?.(
         snapExecutionServiceArgs,
-      ) || new IframeExecutionService(snapExecutionServiceArgs);
+      ) || new WebWorkerExecutionService(snapExecutionServiceArgs);
```

Comment:
This needs to be updated to use @ritave's build types work, so we can create a separate build for iframes and web workers.

Reply:
- #18125.